tests/Type/Nette/data/json_decode.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Misses namespace

Reply:
Not really, this is just being parsed by test processor.